graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/DataTypeGenerator.kt

```diff
@@ -96,15 +103,15 @@ class DataTypeGenerator(private val config: CodeGenConfig, private val document:
                     }
                 )
 
-            return generate(name, unionTypes + implements, fieldDefinitions, config.implementSerializable, definition.description, config.generateAllConstructor)
+            return generate(name, unionTypes + implements, fieldDefinitions, definition.description)
```

Comment:
There is no reason to pass `implementSerializable` or `generateAllConstructor` to the `generate` method since the `config` element is already owned by the instance.